### ISSUE TITLE
Would like to connect multiple issues in a PR

### ISSUE THREAD

**Issue by sfitts** [waffleio/waffle.io#2347]
There are many cases where I may work on more than one issue in a branch.  When that happens I would like to be able to reference them in the PR description (via the closes keyword) and have waffle move the cards for all of the referenced issues.

If this is supposed to work, it doesn't seem to.  For me it only moved the first issue I referenced and not the others.

**Comment by kellihsf** [comment]
hi @sfitts,

You'll need to use the language 'Closes #14, closes #12, closes #13' within the PR title or description for it to close multiple cards.  One thing to note is that you must have the 'closes #13' in the title or description of the pull request at the time you issue the pull request (comments are not currently supported.)  Also, the issue won't close until that pull request is merged into your default branch (usually master, depending on your workflow).

If you have any more questions on this, feel free to email us at support@waffle.io. Thanks!

**Comment by sfitts** [comment]
Thanks for the reply.  I'm pretty sure I did exactly as you suggest (referenced the issues via the "closes" keyword in the PR description), but it only closed the first issue that I referenced.  The next time I have the occasion to give this a try, I'll let you know how it goes.

**Comment by kellihsf** [comment]
Great, sounds like a plan! :)

**Comment by sfitts** [comment]
I have what I think is a related question.  We have a column in which pending PRs go (ready to review).  If I create a PR which references multple issues as you describe (using closes) should it move all of the referenced issues to the ready to review column?

**Comment by NachoCozza** [comment]
@kellihsf does this work? I think it'd be pretty useful, as in our team we separate a feature into multiple issues, and when the whole feature is done, there is no way to group all the issues in a single PR

**Comment by kellihsf** [comment]
@NachoCozza, 
I think the wording may have been confusing in the first response. You can close multiple issues with a single PR, but the issues won't all be connected to each other. If you opened PR #2, and use the language 'closes #4, closes #5, closes #6' in the PR title/description, you'll see that PR #2 would be 'nested' under each of these issue individually, they wouldn't be combined into one unit. Once the PR is merged, they'll all be closed still staying separate. So to sum up, no, it isn't possible to group multiple issues into a single PR.
